tests/constant_time/kem/passes.json

```diff
@@ -1,6 +1,7 @@
 {
-  "BIKE-L1": [],
-  "BIKE-L3": [],
+  "BIKE-L1": ["bike"],
+  "BIKE-L3": ["bike"],
+  "BIKE-L3": ["bike"],
```

Comment:
This should read "BIKE-L5". Tests otherwise fail.

Reply:
right, fixed it.